walkeeper/src/bin/safekeeper.rs

```diff
@@ -54,7 +54,7 @@ fn main() -> Result<()> {
             Arg::with_name("ttl")
                 .long("ttl")
                 .takes_value(true)
-                .help("interval for keeping WAL as walkeeper node, after which them will be uploaded to S3 and removed locally"),
+                .help("interval for keeping WAL as safekeeper node, after which them will be uploaded to S3 and removed locally"),
```

Comment:
since this line is changed, s/as/at, though this implementation is very raw anyway...